Doc/whatsnew/3.12.rst

```diff
@@ -98,6 +98,61 @@ Deprecated
 ==========
 
 
+Pending Removal in Python 3.13
+==============================
+
+The following APIs have been deprecated in earlier Python releases,
+and will be removed in Python 3.13.
+
+Removed :pep:`594` modules:
+
+* :mod:`aifc`
+* :mod:`audioop`
+* :mod:`cgi`
+* :mod:`cgitb`
+* :mod:`chunk`
+* :mod:`crypt`
+* :mod:`imghdr`
+* :mod:`mailcap`
+* :mod:`msilib`
+* :mod:`nis`
+* :mod:`nntplib`
+* :mod:`ossaudiodev`
+* :mod:`pipes`
+* :mod:`sndhdr`
+* :mod:`spwd`
+* :mod:`sunau`
+* :mod:`telnetlib`
+* :mod:`uu`
+* :mod:`xdrlib`
+
+Other removed APIs:
```

Comment:
I think this wording could be improved, suggestion below:
```suggestion
Other APIs pending removal:
```

Another option would be to bring the initial paragraph down to here, and add a little more bridging text for removed modules. (I think this is a better option, but harder to do an inline suggestion for!)

A

Reply:
The opening paragraph could say `The following modules and APIs ...` and then there could simply be two lists:
```rst
The following modules and APIs have been deprecated in earlier
Python releases, and will be removed in Python 3.13.

Modules (see :pep:`594`):
* :mod:`aifc`
* :mod:`audioop`
* ...

APIs:
* :class:`configparser.LegacyInterpolation` (:issue:`46607`)
* :func:`locale.getdefaultlocale` (:issue:`46659`)
* ...
```

It might also be better to list them inline to keep the whole section more compact:
```rst
The following modules and APIs have been deprecated in earlier
Python releases, and will be removed in Python 3.13.

Modules (see :pep:`594`): :mod:`aifc`, :mod:`audioop`, ...

APIs: :class:`configparser.LegacyInterpolation` (:issue:`46607`),
:func:`locale.getdefaultlocale` (:issue:`46659`), ...
```

Reply:
List:

<img width="805" alt="image" src="https://user-images.githubusercontent.com/1324225/171050742-de57c852-5e02-43ed-a0fc-76ec254e602b.png">

Single line:

<img width="819" alt="image" src="https://user-images.githubusercontent.com/1324225/171051036-a5f37a8b-a3f8-4130-91aa-d257292b3492.png">

I prefer the list in both cases and especially think the single line of APIs isn't very readable, but the single line of modules is okay.

Reply:
Thanks for the preview Hugo, the list is significantly clearer for both cases in my opinion.

A

Reply:
> I prefer the list in both cases and especially think the single line of APIs isn't very readable, but the single line of modules is okay.

I agree.  For the sake of consistency I'm fine with using a list for both.